website/content/en/docs/building-operators/golang/advanced-topics.md

```diff
@@ -285,6 +285,64 @@ func main() {
 
 When the operator is not running in a cluster, the Manager will return an error on starting since it can't detect the operator's namespace in order to create the configmap for leader election. You can override this namespace by setting the Manager's `LeaderElectionNamespace` option.
 
+### Multiple architectures
+
+Operators may run on various architectures: x86_64, aarch64, ppc64le, s390x, etc.
+
+#### Manifest lists
+
+The most straightforward way of building operators and operands supporting multiple architectures is to leverage manifest lists, specified by [Image Manifest V2, Schema 2](https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list) or [OCI Image Index](https://github.com/opencontainers/image-spec/blob/main/image-index.md). A manifest list points to specific image manifests for one or more platforms.
+
+For convenience tools like [buildah](https://github.com/containers/buildah/blob/main/docs/buildah-bud.md#building-an-multi-architecture-image-using-a---manifest-option-requires-emulation-software) allow to cross-build and manifest multi-arch containers on one host. For instance:
+
+```shell
+for a in amd64 arm64 ppc64le s390x; do \
+  buildah bud --manifest registry/username/repo:v1 --arch $a; \
+done
+```
+
+This creates the manifest list, builds each image and adds them to the manifest list.
+
+The result can then be pushed to the desired registry.
+
+```shell
+buildah push registry/username/repo:v1
+```
+
+Docker with [buildx](https://docs.docker.com/buildx/working-with-buildx/#build-multi-platform-images) provides similar capabilities.
+
+```shell
+docker buildx build --push --platform linux/amd64,linux/arm64,linux/ppc64le,linux/s390x --tag registry/username/repo:v1 .
+```
+
+See [docker documentation](https://docs.docker.com/buildx/working-with-buildx/#build-multi-platform-images) for additional options.
+ 
+**Caveats**: the Dockerfile generated by the SDK for the operator explicitely references GOARCH=amd64 for go build. This can be amended to GOARCH=$TARGETPLATFORM. Docker will automatically set the environment variable to the value specified by --platform. With buildah --build-arg will need to be used for the purpose.
+
+All image references in the ClusterServiceVersion should be manifest lists containing the pointers to the image manifests for the supported architectures.
+
+#### Architecture specific content
+
+Operand images that require different image content (such as unique runtime packages) per architecture should be handled by the operator itself based on environment variables. That is, the CSV can list environment variables for each supported architecture / OS, for example:
+
+```
+MY_OPERAND_ARM64: my/arm64/image
+MY_OPERAND_PPC64LE: my/ppc64le/image
+```
```

Comment:
Thinking about this more, can't this be accomplished with manifest lists only? The arch-specific images that a manifest list points to can be completely different (in fact, they already are since they have different platform-specific binaries).